### PR TITLE
Add missing deps

### PR DESCRIPTION
**Pull request by ptrfrncsmrph**
**Description of the change**

This adds missing dependencies listed when running `spago build` (see: https://github.com/purescript-contrib/governance/issues/43)

<details><summary>Output from running <code>spago build</code> before these changes</summary>

<pre>
[info] Build succeeded.
[warn] None of your project files import modules from some projects that are in the direct dependencies of your project.
These dependencies are unused. To fix this warning, remove the following packages from the list of dependencies in your config:
- free
[error] Some of your project files import modules from packages that are not in the direct dependencies of your project.
To fix this error add the following packages to the list of dependencies in your config:
- arrays
- bifunctors
- control
- either
- foldable-traversable
- maybe
- newtype
- prelude
- refs
- tailrec
You may add these dependencies by running the following command:
spago install arrays bifunctors control either foldable-traversable maybe newtype prelude refs tailrec

</pre>

</details>

---

**Checklist:**

- [x] Added the change to the changelog's "Unreleased" section with a link to this PR and your username
- [x] Linked any existing issues or proposals that this pull request should close
- [ ] Updated or added relevant documentation in the README and/or documentation directory
- [ ] Added a test for the contribution (if applicable)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ New features:
 
 Bugfixes:
 - Fixed typo in `forkAff` documentation (#184)
+- Updated missing dependencies in `spago.dhall` (@ptrfrncsmrph) (#205)
```
Comment (thomashoneyman) [inline CHANGELOG.md:25]: This should go under the "Unreleased" section

**Comment by thomashoneyman** [comment]
This looks good, though it's now revealing some unused name errors -- would you mind taking a look at those? Thank you!

**Comment by ptrfrncsmrph** [comment]
Sorry for the force push, got confused which repo I was in 😅

**Comment by thomashoneyman** [comment]
No worries 😆 Also, I don't know if you saw it yet, but I updated the original issue to recommend not worrying about the bower.json files. They're a bit of a mess and you can just leave them as-is. I can tweak them as needed before publishing the library; they aren't used in local development anymore.

**Pull request merged**